src/webui/www/private/scripts/dynamicTable.js

```diff
@@ -610,23 +610,23 @@ var DynamicTable = new Class({
 
         row['data'] = data;
 
-        for (var x in data)
+        for (const x in data)
             row['full_data'][x] = data[x];
     },
 
     getFilteredAndSortedRows: function() {
-        var filteredRows = [];
+        const filteredRows = [];
```

Comment:
What does const mean in JavaScript? It is modified in code below...

Reply:
It means we can't change variable's value, the reference to array object instance. But we are free to call any of its methods or change its properties. There is no notion of const methods in JS.